File: ejerciciosMariet/Java-ejercicios/Test3.java
import java.util.Scanner;

public class Test3 {
    public static void main(String[] args) {
    	
        Scanner scanner = new Scanner(System.in);

        int opciones;
      
    do {

    	System.out.println("Ingrese una opcion del 1 al 4");
        opciones = scanner.nextInt();
       switch (opciones) {
            case 1:
            Nombre_Apellido (scanner);
                break;
            case 2:
                correo_contraseña(scanner);
                break;
            case 3:
                usuario(scanner);
                break;
            case 4:
                System.out.println("Bye bye sapo");
                break;
            default:
                System.out.println("ERROR, prueba con otra opcion");
                break;
            
        } 
    	  
       
    }  
       
    while(opciones!=4); {
      
    	scanner.close();
    }
} 
      
    public static void Nombre_Apellido(Scanner scanner) {
    	
    	System.out.print("Truco ");
        String Truco = scanner.nextLine();
        
        System.out.print("Ingrese nombre: ");
        String nombre = scanner.nextLine();
        System.out.print("Ingrese apellido: ");
        String apellido = scanner.nextLine();
    }

    public static void correo_contraseña(Scanner scanner) {
    	
    	System.out.print("Truco ");
        String Truco = scanner.nextLine();
        
        System.out.print("Ingrese email: ");
        String email = scanner.nextLine();
        System.out.print("Ingrese contraseña: ");
        String contraseña = scanner.nextLine();
    }

    public static void usuario(Scanner scanner) {
        System.out.print("Ingrese el tipo de usuario (1 para administrador, 2 para cliente): ");
        int tipoUsuario = scanner.nextInt();
        if (tipoUsuario == 1) {
            System.out.println("Eres administrador");
        } else if (tipoUsuario == 2) {
            System.out.println("Eres cliente");
        } else {
            System.out.println("Invalido");
        }
    }
}

File: ejerciciosMariet/Python-ejercicios/test3.py
def nombre_apellido():
    truco = input("Truco: ")
    nombre = input("Ingrese nombre: ")
    apellido = input("Ingrese apellido: ")

def correo_contraseña():
    truco = input("Truco: ")
    email = input("Ingrese email: ")
    contraseña = input("Ingrese contraseña: ")

def usuario():
    tipo_usuario = int(input("Ingrese el tipo de usuario (1 para administrador, 2 para cliente): "))
    if tipo_usuario == 1:
        print("Eres administrador")
    elif tipo_usuario == 2:
        print("Eres cliente")
    else:
        print("Inválido")

opciones = 0

while opciones != 4:
    print("Ingrese una opción del 1 al 4")
    opciones = int(input())
    
    if opciones == 1:
        nombre_apellido()
    elif opciones == 2:
        correo_contraseña()
    elif opciones == 3:
        usuario()
    elif opciones == 4:
        print("Bye bye sapo")
    else:
        print("ERROR, prueba con otra opción")